docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
+- Django >= 1.7.10, < 1.8
+- Translate Toolkit >= 1.13.0
+- Python >= 2.7, < 3.0
+- Redis >= 2.8.4
+- Django transaction hooks
+- Unix-based operating system.
+
+
+Major Changes
+=============
+
+- Pulled latest translations.
+- MyISAM support has been dropped.
+- The editor for static pages now highlights the content's markup and displays
+  a live preview.
+- Added support for Elasticsearch-based external Translation Memory servers.
+- Cleaned up connection logic and added checks for misconfigured Translation
+  Memory servers.
+
+Below we provide much more detail. These are by no means exhaustive, view the
+`git log <https://github.com/translate/pootle/compare/2.7.2...2.7.3>`_ for
+complete information.
+
+
+Details of changes
+==================
+
+- Pulled latest translations.
+- Static pages and announcements:
+
+  - The editor for static pages now highlights the content's markup and displays
+    a live preview of the rendered contents (:issue:`3346`, :issue:`3766`).
+  - Project and language announcements are now also displayed on their
+    respective overview pages.
+
+- Translation memory:
+
+  - :djadmin:`update_tmserver`:
+
+    - Renamed :option:`--overwrite` to :option:`--refresh`.
+    - Translations from disabled projects must be explicitly included with
+      :option:`--include-disabled-projects`.
+    - Added support for Elasticsearch-based external Translation Memory
+      servers, which can be populated from translation files or directories on
+      disk. This effectively brings the ability to display TM results from
+      different TM servers, sorting them by their score.
+    - Translations saved from Pootle now include a timestamp.
+    - Fixed missing index error :issue:`4120`.
+
+  - :setting:`POOTLE_TM_SERVER`:
+
+    - The ``default`` TM server has been renamed to ``local``. Make sure to
+      adjust your settings.
+    - Added a new :setting:`WEIGHT <POOTLE_TM_SERVER-WEIGHT>` option to raise
+      or lower the TM results score for each specific TM server.
+    - Added several checks to ensure this setting is not misconfigured.
+
+  - Changed connection logic for Translation Memory servers
+
+- Database:
+
+  - `InnoDB <https://dev.mysql.com/doc/refman/5.6/en/innodb-storage-engine.html>`_
+    is the only accepted MySQL backend. Deployments using MyISAM must
+    :doc:`migrate to either MySQL (InnoDB) or PostgreSQL </server/database_migration>`.
+  - Close a database connection before and after each rqworker job once it
+    exceeds the maximum age to imitate Django's request/response cycle
+    :issue:`4094`.
+
+- Editor:
+
+  - Non-critical checks can be now muted/unmuted.
```

Comment:
again? I am not 100% sure it was possible to mute/unmute non-critical checks before.